src/component/list.js

```diff
@@ -117,6 +124,76 @@ ListHeader.propTypes = {
   style: View.propTypes.style,
 };
 
+//
+// Setting Content
```

Comment:
Maybe we should move all the `Setting` components into their own component file?